pkg/operator/ceph/csi/csi.go

```diff
@@ -84,7 +84,7 @@ func (r *ReconcileCSI) setParams(ver *version.Info) error {
 	}
 	CSIParam.EnableCSIGRPCMetrics = fmt.Sprintf("%t", EnableCSIGRPCMetrics)
 
-	if CSIParam.EnableCSIHostNetwork, err = strconv.ParseBool(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_HOST_NETWORK", "true")); err != nil {
+	if CSIParam.EnableCSIHostNetwork, err = strconv.ParseBool(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_HOST_NETWORK", defaultHostNetwork)); err != nil {
```

Comment:
Can we log that hostNetworking is not recommended when deploying the CSI driver? Perhaps as a warning.